src/v2/coins/ofcToken.js

```diff
@@ -0,0 +1,68 @@
+const Ofc = require('./ofc');
+const _ = require('lodash');
+const Promise = require('bluebird');
+const co = Promise.coroutine;
+const prova = require('prova-lib');
+
+class OFCToken extends Ofc {
+
+  static get tokenConfig() {
+    return {};
+  }
+
+  constructor() {
+    super();
+    Object.assign(this, this.constructor.tokenConfig);
```

Comment:
What does this do?

Reply:
This will, upon creation of an OFCToken object, copy the token config properties onto the newly created token object.